src/cryptonote_core/service_node_list.cpp

```diff
@@ -5036,20 +5107,29 @@ bool service_node_list::store(uint64_t state_height) {
         cryptonote::db_wtxn_guard txn_guard{db};
 
         if (m_transient->long_term_data_dirty) {
-            TracyCZoneN(serialize_step, "Serialize archive array of blobs", true);
+            TracyCZoneN(serialize_step, "Serialize and compress archive array of blobs", true);
             serialization::binary_string_archiver ar;
+
             std::string db_blob = serialize_db_blob(ar, archive_blob_list, nullptr);
+            db_blob = zstd_compress(
```

Comment:
The first parameter in `zstd_compress` is `std::string_view` so the `std::move` can be removed.

Reply:
Resolved.  The whole first version of `db_blob` here isn't needed (and was only like this because there was a debug line in between these two lines when I was working on this).